tests/playwright/assets/api-requests.ts

```diff
@@ -234,4 +258,36 @@ export default class ApiRequests {
 
 		return response;
 	}
+
+	public async createNewUser( request: APIRequestContext, user ) {
```

Comment:
Assuming there is a `User` type, if not, I think we should create one
```suggestion
	public async createNewUser( request: APIRequestContext, user: User ) {
```